api.ts

```diff
@@ -0,0 +1,73 @@
+import { MinorityGroups, BusinessType } from "./src/models";
```

Comment:
I think this is a duplicate file of `./constants/enumvonverters`

Reply:
oh yeah i added that white i was waiting for fin’s branch to get be merged but i can delete it